packages/snowpack/src/config.ts

```diff
@@ -493,6 +493,9 @@ function normalizeAlias(config: SnowpackConfig, createMountAlias: boolean) {
       replacement.startsWith('/')
     ) {
       cleanAlias[target] = path.resolve(cwd, replacement);
```

Comment:
I generally prefer to normalize data for config, so lets just do:

```suggestion
      cleanAlias[addTrailingSlash(target)] = addTrailingSlash(path.resolve(cwd, replacement));
```

That would also solve the issue of "@app" matching any import that starts with "@app", and not "@app/*" (which was the intention.)

Reply:
Yeah, agree with U.